bin/oay/Cargo.toml

```diff
@@ -51,7 +51,7 @@ dav-server-opendalfs = { version = "0.0.1", path = "../../integrations/dav-serve
 dirs = "5.0.1"
 futures = "0.3"
 futures-util = { version = "0.3.29", optional = true }
-opendal = { version = "0.45.1", path = "../../core" }
+opendal = { version = "0.46.0", path = "../../core" }
```

Comment:
oay itself should be bumped to next patch version. Or do you prefer to do them in separate PR?

Reply:
This is the version of opendal, and there is no 0.45.2 to be released. What do you mean here? At least you comment at the wrong line.